tests/Makefile

```diff
@@ -0,0 +1,35 @@
+
+test-setup: setup-modules setup-config
+
+test-setup-phpunit: setup-modules setup-config-phpunit
+
+# Install a test site.
+install-site:
+	drush site-install ding2
+
+# Setup test modules on site.
+setup-modules:
+	# Nothing to be done yet.
+
+setup-config:
+	# Configure ting.module
+	drush vset --exact ting_agency 100200
+	drush vset --exact ting_search_url 'http://oss-services.dbc.dk/opensearch/4.0.1/'
+	drush vset --exact ting_search_profile test
+
+setup-config-phpunit: setup-config
+	# ting.module.
+	drush vset --exact ting_agency '775100'
+	drush vset --exact ting_search_profile 'ding2taltest'
+	drush vset --exact ting_search_url 'http://opensearch.addi.dk/4.0.1/'
```

Comment:
It is, however that might be the reason that there is no items available - since this URL is restricted by IPs.
Similar to the issue you have with moreinfo.